CMakeLists.txt

```diff
@@ -281,3 +281,38 @@ if (ZYDIS_BUILD_TOOLS AND NOT ZYAN_NO_LIBC)
         #_maybe_set_emscripten_cfg("ZydisPE")
     endif ()
 endif ()
+
+//------------for kernel
+cmake_minimum_required(VERSION "3.15")
+project("zydisRing0")
+set(CMAKE_CXX_STANDARD 17)
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/../FindWdk")
+find_package(WDK REQUIRED)
+option(ZYAN_NO_LIBC ON)
+option(ZYDIS_FEATURE_FORMATTER "" ON)
+option(ZYDIS_BUILD_TOOLS "" OFF)
+option(ZYDIS_BUILD_EXAMPLES "" OFF)
+add_subdirectory("zydis")#没有这个不能生成配置文件 ,先用这个生成配置文件，debug目录下copy到跟目录
+include_directories(${CMAKE_CURRENT_LIST_DIR})
```

Comment:
Please don't merge this as it could break any CMake project that depends on Zydis

Reply:
No worries @mrexodia . We don't plan to merge it in the current form 😄 We just left this open as a reminder for the future.